src/main/kotlin/com/hibob/project/service/feedbackService.kt

```diff
@@ -0,0 +1,23 @@
+package com.hibob.project.service
+
+import com.hibob.project.dao.*
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class FeedbackService @Autowired constructor(private val feedbackDao: FeedbackDao) {
+    fun getAllFeedbacks(companyId: Long): List<FeedbackConfiguration> {
+        val feedbacks = feedbackDao.getAllFeedbacks(companyId)
+        return feedbacks
+    }
+
+    fun feedbackSubmission(feedbackCreationRequest: FeedbackCreationRequest, employee: Employee) {
+        if (feedbackCreationRequest.text.length < 2)
+            throw IllegalArgumentException("Text must be at least 2 characters")
+
```

Comment:
no blank line between if and else if, it should be clear that it is the same code block

Reply:
think of the function structure, it's called submit but it does validation and submit. Most of the code here is validation and if you need to add more validations, the code will become longer.
So think of the function structure here